src/mlpack/tests/async_learning_test.cpp

```diff
@@ -1,6 +1,7 @@
 /**
  * @file async_learning_test.hpp
  * @author Shangtong Zhang
+ * @author Rohan Raj
```

Comment:
I think we should only add the ``@author`` tag if someone made a reasonable contribution to the file; every contribution is very much appreciated, but I think it would be fair if we add names according to the rule I have mentioned. Let me know what you think.